src/reporters.ts

```diff
@@ -124,7 +124,15 @@ export class HumanReporter extends QueryReporter {
         (field: string) =>
           (formattedColumns[field] = {
             header: field.toUpperCase(),
-            get: (row): string => get(row, field) as string,
+            get: (row): string => {
```

Comment:
Modified the get to first check if the field, as given, can be referenced directly, This solves case where property has a name like `foo.bar`. If cannot be directly referenced then delegate to `get`